python/lsst/rubintv/production/utils.py

```diff
@@ -791,7 +791,9 @@ def getShardedData(path,
                                            seqNum=seqNum)
 
     start = time.time()
-    while time.time() - start < timeout:
+    firstLoop = True
+    while firstLoop or time.time() - start < timeout:
```

Comment:
I generally prefer having lengthier conditions in parentheses like `(time.time() - start < timeout)` for clarity, even if it's not necessary. YMMV.

Reply:
Totally agree 👍